app/code/Magento/InventoryConfigurableProduct/view/adminhtml/web/js/components/dynamic-rows-configurable.js

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
Also, in general, I think that in current case extends DR component it is ain't better solution. You can make the separate component that will listen to DynamicRows state and manage matrix and sources state.

Reply:
Done.